PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3328,6 +3328,19 @@ void Notepad_plus::checkModifiedDocument(bool bCheckOnlyCurrentBuffer)
 	MainFileManager->checkFilesystemChanges(bCheckOnlyCurrentBuffer);
 }
 
+void Notepad_plus::checkModifiedDocumentIfMonitoring()
+{
+	// This is an workaround to deal with Microsoft issue in ReadDirectoryChanges notification
+	// If command prompt is used to write file continuously (e.g. ping -t 8.8.8.8 > ping.log)
+	// Then ReadDirectoryChanges does not detect the change
+	// Fortunately, notification is sent if right click or double click happens on that file
+	// Let's leverage this as workaround to enhance npp file monitoring functionality.
+
+	Buffer* currBuf = MainFileManager->getCurrentBuffer();

```

Comment:
Use getCurrentBuffer() of class Notepad_plus instead.

Reply:
You're right. I didn't notice it.

Reply:
Fixed it now.